Automattic/wp-calypso
# Improve Social Signup: Send directly to password if existing email

## Current

Visit `https://wordpress.com/start/user-social`, insert an already existing email.
You will see an error message, and if you click on `Log in now`, you will be redirected to the log-in page.

## Now

With this PR, if the email already exists, when the user inserts the email and clicks, it gets directly redirected to the log-in page. We show the user a notice.

We excluded passwordless accounts in this PR (no reason to send them to password..)

Added `error_code` (can be 'already_taken', 'already_active', 'email_exists' or others) to `calypso_signup_actions_onboarding_passwordless_login_error` event.

## Why

If the user inserts an existing email, it's clear the intent to use that to enter WordPress.com. We should remove friction when it is possible.


https://github.com/Automattic/wp-calypso/assets/17054134/6f61ec64-084a-488f-b3d6-9d2fae8ebed8

## Translation

![image](https://github.com/Automattic/wp-calypso/assets/52076348/38b2bf5e-a2cd-483c-9398-86cfc68caae5)


## Testing

1. Live Link
2. Visit `/start`
3. Click Email and insert an existing email
4. Click on `Continue`
5. You should land in /log-in with the ability to directly insert the password. You should also see a notice.
6. Check for translations.
